consumer-rest-core/src/main/scala/za/co/absa/spline/consumer/rest/controller/ExecutionEventsController.scala

```diff
@@ -17,26 +17,27 @@ package za.co.absa.spline.consumer.rest.controller
 
 import java.lang.System.currentTimeMillis
 
-import io.swagger.annotations.{ApiOperation, ApiParam}
+import io.swagger.annotations.{Api, ApiOperation, ApiParam}
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.web.bind.annotation._
 import za.co.absa.spline.consumer.service.model._
 import za.co.absa.spline.consumer.service.repo.ExecutionEventRepository
 
 import scala.concurrent.Future
 
-@RestController("execution-event")
-class ExecutionEventController @Autowired()(val repo: ExecutionEventRepository) {
+@RestController
+@Api(tags = Array("execution-events"))
+class ExecutionEventsController @Autowired()(val repo: ExecutionEventRepository) {
 
   import scala.concurrent.ExecutionContext.Implicits._
 
-  @GetMapping(Array("/executionEvent"))
+  @GetMapping(Array("/execution-events"))
   @ApiOperation(
-    value = "GET /executionEvent",
-    notes = "Returns a Pageable list of execution events within the time range given in parameters",
+    value = "Get execution events",
+    notes = "Returns a pageable list of execution events filtered by the query parameters",
     response = classOf[PageableExecutionEventsResponse]
   )
-  def executionEvent
+  def executionEvents
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The number of parameters should not exceed 8](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4536902)

Reply:
codacy, you must be kidding!